test/Round-test.js

```diff
@@ -1,24 +1,22 @@
 const chai = require('chai');
 const expect = chai.expect;
+const data = require('../src/data');
+const prototypeQuestions = data.prototypeData;
 
 const Round = require('../src/Round');
 const Card = require('../src/Card');
 const Deck = require('../src/Deck');
 const { it } = require('mocha');
 
 describe('Round', () => {
-  let card1;
-  let card2;
-  let card3;
   let cards;
   let deck;
   let round;
 
   beforeEach(() => {
-    card1 = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
-    card2 = new Card(2, "What's a food?", ['object', 'array', 'pizza'], 'pizza');
-    card3 = new Card(3, "What's my favorite food?", ['pizza', 'sushi', 'oatmeal'], 'sushi');
-    cards = [card1, card2, card3];
+    cards = prototypeQuestions.map(data => {
+      return new Card(data.id, data.question, data.answers, data.correctAnswer);
```

Comment:
Iterating Ithrough the data-model to create multiple cards makes is far simpler to populate my test with data, and more accurately represents use-cases